eng/Versions.props

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <!-- Production Dependencies -->
   <PropertyGroup>
-    <PrivateSourceBuildReferencePackagesPackageVersion>1.0.0-beta.19514.1</PrivateSourceBuildReferencePackagesPackageVersion>
-    <PrivateSourceBuiltArtifactsPackageVersion>0.1.0-3.0.100</PrivateSourceBuiltArtifactsPackageVersion>
+    <PrivateSourceBuildReferencePackagesPackageVersion>1.0.0-beta.19554.1</PrivateSourceBuildReferencePackagesPackageVersion>
+    <PrivateSourceBuiltArtifactsPackageVersion>0.1.0-3.1.100-preview3</PrivateSourceBuiltArtifactsPackageVersion>
```

Comment:
Does this version exist?  I only see 3.1.100.

Reply:
Nope, I thought this was the output version.  Reverted.